code/controllers/subsystem/traumas.dm

```diff
@@ -169,6 +170,11 @@ SUBSYSTEM_DEF(traumas)
 			/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom
 		)),
 
+		"guns" = typecacheof(list(/obj/item/gun, /obj/item/ammo_casing/, /obj/machinery/porta_turret, /obj/machinery/power/emitter,
```

Comment:
split the obj item gun to energy and ballistic guns so it doesnt count wands, also you have a / after ammo_casing

Reply:
This feels more like an example of things being made guns when they're not actually guns.
Wands, staffs, meathooks, and bonechillers are all examples of that.

Just energy and ballistic isn't encompassing enough imo.